src/main/java/seedu/address/model/person/Person.java

```diff
@@ -105,13 +118,14 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this)
+        ToStringBuilder builder = new ToStringBuilder(this)
                 .add("name", name)
                 .add("phone", phone)
-                .add("email", email)
-                .add("address", address)
-                .add("tags", tags)
-                .toString();
+                .add("email", email);
+        address.ifPresent(addr -> builder.add("address", this.getAddress()));
```

Comment:
sorry one more small nit, this can be `address.ifPresent(addr -> builder.add("address", addr));` instead